lib/MarathonEventBusClient.js

```diff
@@ -101,7 +108,9 @@ MarathonEventBusClient.prototype.subscribe = function () {
         url = self.options.marathonProtocol + "://" + self.options.marathonHost + ":" + self.options.marathonPort+self.options.marathonUri;
 
     // Allow unauthorized HTTPS requests
-    let eventSourceInitDict = { rejectUnauthorized: false };
+    let eventSourceInitDict = self.options.marathonHeaders;
```

Comment:
cool that sounds good and yeah maybe didnt wrote my code right at it does not looks that i have th header property on the object but i'll change that in the next commit.